visivo/models/base/parent_model.py

```diff
@@ -153,19 +153,92 @@ def name_match(trace):
 
         return list(filter(name_match, objects))
 
-    @staticmethod
-    def show_dag(dag):
-        import matplotlib.pyplot as pyplot
-        from networkx import spring_layout, draw_networkx
-
-        options = {}
-        pos = spring_layout(dag)
-        draw_networkx(dag, pos, **options)
-
-        ax = pyplot.gca()
-        ax.margins(0.20)
-        pyplot.axis("off")
-        pyplot.show()
+    def show_dag(self, dag=None):
```

Comment:
This is cool, are you thinking of including this chart in the viewer by default?

Reply:
Not yet, but eventually.  If we are using plotly, lets start going that way!